src/components/block/index.ts

```diff
@@ -630,7 +655,7 @@ export default class Block extends EventsDispatcher<BlockEvents> {
     const defaultTunesElement = document.createDocumentFragment();
 
     this.tunesInstances.forEach((tune) => {
-      $.append(tunesElement, tune.render());
+      $.append(tunesElement, tune.render(SelectionObserver.getSelectedRange()));
```

Comment:
do not pass range to the Block Tunes constructor, because the Range is not related to the Block Tunes